embedded/document/engine_test.go

```diff
@@ -45,6 +41,7 @@ func makeEngine(t *testing.T) *Engine {
 	return engine
 }
 
+/*
```

Comment:
In case this is commented out by mistake

Reply:
thanks, need to re-enable them yet